.gitattributes

```diff
@@ -14,3 +14,6 @@
 /phpunit.xml.dist   export-ignore
 /test               export-ignore
 /docs               export-ignore
+.phpstorm.meta.php  export-ignore
+phpcs.xml.dist      export-ignore
+phpstan.neon.dist   export-ignore
```

Comment:
[`.phpstorm.meta.php` provides stubs for auto-complete in PhpStorm](https://www.jetbrains.com/help/phpstorm/ide-advanced-metadata.html) so we'll want to continue including that in our releases.  But the other two should definitely be excluded - thanks for catching those!